src/muse/mca.py

```diff
@@ -64,8 +64,12 @@ def factory(cls, settings: Union[Text, Path, Mapping, Any]) -> MCA:
             ).sel(region=settings.regions)
         ).interp(year=settings.time_framework, method=settings.interpolation_mode)
 
-        market["supply"] = zeros_like(market.exports)
-        market["consumption"] = zeros_like(market.exports)
+        market["supply"] = zeros_like(market.exports).drop_vars(
+            ["timeslice", "month", "day", "hour"]
+        )
+        market["consumption"] = zeros_like(market.exports).drop_vars(
+            ["timeslice", "month", "day", "hour"]
+        )
```

Comment:
We need to drop them because both the `market` dataset and the new `DataArray` we are adding contain those coordinates. In the `old times`, xarray manipulation dealt with that under the hood, but not anymore - at least not for `MultiIndex` coordinates, like `timeslice` - and the conflicting coordinates need to be dropped before assigning them. 

I'm not sure if a helper function would be that helpful. I guess it would be something like:

```python
def drop_timeslice(data: xr.Dataset | xr.DataArray) -> xr.Dataset | xr.DataArray:
    return data.drop_vars(["timeslice", "month", "day", "hour"])
```

Do you think it would be helpful?

Reply:
I think it would probably be a lot of work for not much gain :stuck_out_tongue:. It just feels a bit messy having the repetition, but I guess that's just to do with the code design and maybe it's just what we have to do.